### PR TITLE
Fix Envision visualization issue in "near real time" mode

### PR DESCRIPTION
**Pull request by qianyi-sun**

### CODE REVIEW

File: envision/web/src/components/simulation.js [hunk 1]
```diff
@@ -185,10 +185,11 @@ export default function Simulation({
           // React to trigger update, 0.1 millisecond
           await sleep(0.1);
         } else if (playingMode == PLAYMODES.near_real_time) {
-          await sleep(
-            msInSec * (elapsed_times[0] - prevElapsedTime) -
-              (Date.now() - waitStartTime)
-          );
+          let delt_elapsed_time_in_sec = msInSec * (elapsed_times[0] - prevElapsedTime);
```
Comment (saulfield) [inline envision/web/src/components/simulation.js:188]: This is in milliseconds I believe. Also, I think the style is snakeCase (even though not everything here follows it).
```suggestion
          let deltaElapsedMs = msInSec * (elapsed_times[0] - prevElapsedTime);
```

File: envision/web/src/components/simulation.js [hunk 2]
```diff
@@ -185,10 +185,11 @@ export default function Simulation({
...
+          if (delt_elapsed_time_in_sec > 100) {
+            delt_elapsed_time_in_sec = 100
```
Comment (saulfield) [inline envision/web/src/components/simulation.js:190]: After using Python, it's easy to forget about semicolons. Though not always necessary in js, they should be used to make sure statements stay separate from each other.
```suggestion
            delt_elapsed_time_in_sec = 100;
```

**Review by Gamenot — CHANGES_REQUESTED** [review]
Please add a changelog entry.

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ Copy and pasting the git commit messages is __NOT__ enough.
 - Added new video record ultility using moviepy.
 - Added distance check between bubble and vehicle to avoid generating unnecessary cursors.
 - Added `ConfigurableZone` for `Zone` object to types which enable users to build bubble by providing coordinates of the polygon.
+- Added "SMARTS Performance Benchmark" development tool for evaluating the simulation performance.
```
Comment (Gamenot) [inline CHANGELOG.md:18]: Where did this entry come from?

**Pull request merged**